packages/mask/src/plugins/RedPacket/SNSAdaptor/SelectNftTokenDialog.tsx

```diff
@@ -403,47 +402,49 @@ export function SelectNftTokenDialog(props: SelectNftTokenDialogProps) {
     )
     // #region fetch token detail
     const onSearch = useCallback(async () => {
+        if (!/^(\s?(\d+)?\s?,?)+$/.test(searchTokenListInput)) return
+        const tokenIdList = uniq(searchTokenListInput.split(',').map((v) => Number(v).toString()))
```

Comment:
Does it necessary to assume that numeric token ids?

Reply:
Agree.